subsys/net/lib/app/net_app.c

```diff
@@ -562,27 +562,45 @@ struct net_context *select_client_ctx(struct net_app_ctx *ctx,
 
 #if defined(CONFIG_NET_APP_SERVER)
 #if defined(CONFIG_NET_TCP)
-static struct net_context *get_server_ctx(struct net_app_ctx *ctx,
```

Comment:
Commit message:

~~~
E.g. One net_app_ctx and two net_contexts. 'For' loop goes through
     array of net_contexts and searching for matching context.
     Let's say caller required net_context is second in the
~~~ 

What's the purpose of that intent? Please reformat.

Reply:
Also, that description also doesn't sound too clear to me (but I'm not familiar with net_app_ctx internals, maybe that's it).